test/llvm-intrinsics/uadd.with.overflow.ll

```diff
@@ -0,0 +1,43 @@
+; RUN: llvm-as %s -o %t.bc
+; RUN: llvm-spirv %t.bc -spirv-text -o - | FileCheck --check-prefix CHECK-SPIRV %s
+; RUN: llvm-spirv %t.bc -o %t.spv
+; RUN: spirv-val %t.spv
+; RUN: llvm-spirv -r %t.spv -o %t.rev.bc
+; RUN: llvm-dis %t.rev.bc -o - | FileCheck --check-prefix CHECK-LLVM %s
+
+target triple = "spir64-unknown-unknown"
+
+; CHECK-SPIRV: IAddCarry
```

Comment:
May be we can add tests to make sure the SPIR-V types are correctly generated?
https://registry.khronos.org/SPIR-V/specs/unified1/SPIRV.html#OpIAddCarry

Reply:
Done